### PR TITLE
Rewrite bundle

### LINKED ISSUE #6: Add composer.json (also for postmark repo)
Hi,

Symfony 2.1 RC1 is out and i'd love to test it out. Could you please add the composer.json file?

It's really simple. Something like this: https://raw.github.com/chrisboulton/php-resque/master/composer.json

You specify the miguel250/postmark.git dependency and register your repos on www.packagist.org.

http://packagist.org/about

### PR DESCRIPTION
**Pull request by miguel250**
###### Features
- Remove the need for https://github.com/Znarkus/postmark-php
- Use buzz to handle http requests.
- Easier to update postmark api
- Add composer
- Fix #6
###### Missing:
- Documenting(comments)
- Unit tests
- Batching messages
- Attachments 
###### Example:

``` php
<?php
$message = $this->get('postmark');
$message->addTo('<test@gmail.com>', 'Test');
$message->setSubject('subject');
$message->setTextMessage('email body');
$message->send();
```

### CODE REVIEW

File: Resources/config/services.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+     <parameters>
+        <parameter key="mz_postmark.api_key">null</parameter>
+        <parameter key="mz_postmark.from_email">null</parameter>
+        <parameter key="mz_postmark.from_name">null</parameter>
+        <parameter key="mz_postmark.postmark.main.class">MZ\PostmarkBundle\PostMark\Main</parameter>
+    </parameters>
+
+    <services>
+        <service id="postmark" class="%mz_postmark.postmark.main.class%">
```
Comment (ruudk) [inline Resources/config/services.xml]: Would it be possible to set the service as shared=false? That way, you get a new instance of Postmark\Main when you call 'postmark' service from the DIC.

**Comment by phidah** [comment]
Awesome :)

**Comment by ruudk** [comment]
When will this be ready and merged? :)

**Comment by ruudk** [comment]
@miguel250 Wouldn't it be better to just make different services in the DIC? 

```
<?php
$message = $this->get('postmark.message');
$batch = $this->get('postmark.batch');
$bounces = $this->get('postmark.bounces');
```

I don't like the Uppercased methods you've created.

**Comment by ruudk** [comment]
If you create separate services you can inject the base 'postmark' service to all of them. That service contains the API key etc. 'postmark.message' would then be a single message instance as it returns a new instance on every call.

**Comment by miguel250** [comment]
@ruudk I won't have time to work on this until next week but pull requests are welcome against the rewrite branch.

**Comment by ruudk** [comment]
I also think the "PostMark" directory should be called "Postmark" as that's the official name of the company.

**Pull request merged**